net/pb/lstore.pb.go

```diff
@@ -20,7 +21,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
Is this from using a newer gogo proto package?

Reply:
Oops I overlooked this one. Not sure actually. I'll dig into it

Reply:
It is because I'm using the latest version of `protoc-gen-gogofaster` to generate this piece of code. The purpose here is to raise a compilation error when `github.com/gogo/protobuf/proto` package is too old and doesn't have `GoGoProtoPackageIsVersion3` defined. I think we are good.